packages/core/src/RenderingEngine/GenericViewport/Volume3D/LOADING-brick-volume-source.md

```
@@ -0,0 +1,707 @@
+An anisotropic series has the same layout with spacing-driven level names, and its
+coarsest level is a single object — as actually generated for Juno:
+
+```
+    brick/
+      manifest.json
+      d1/     k000/y0x0.jls … y7x7.jls   # 8x8 per k, k000 … k002   (192)
+      d4_4_1/ k000/y0x0.jls … y1x1.jls   # 2x2 per k, k000 … k002   (12)
+      d8_8_2/ k000/y0x0.jls              # the whole level, 185 KB  (1)
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)